CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features
 
 * **Add parent_element to overlapping case output** Adds parent_element to the output for `identify_overlapping_or_nesting_case` and `catch_overlapping_and_nested_bboxes` functions.
+* **Separate grouping function for evaluation** Separate the grouping functionality to avoid duplicated evaluation when the granular level eval is already available.
```

Comment:
looks like this PR is just moving a block of code to a separate function and i am just curious how this avoid duplicated evaluation?

Reply:
Sorry I wasn't clear on the description. We are adding a metric evaluation for individual docs and aggregated docs after this. The current structure of the code makes it impossible to get the grouping from existing granular output, but rather need to reevaluate. However, that reminds me I'd want to make edits to the command part of the eval as well. Will convert this back to draft!

Reply:
Is this a user-facing function? If not it probably shouldn't have a CHANGELOG text entry. It seems like this is a refactoring to extract a function so we can access it from multiple places. In that case just and updated version string would be sufficient, like `dev8` instead of `dev7` or whatever.